mitodl/ocw-data-parser
# Add black and pylint and run them in github actions

#### What are the relevant tickets?
Fixes #57 

#### What's this PR do?
Adds pylint and black and add steps to run them in github actions. There should not be any functionality changes in the code, and I marked commits with "reformatting" if there are only reformatting related changes.

#### How should this be manually tested?
Convert a course and verify that the output JSON files look right. Nothing should change in the output.
